configure.ac

```diff
@@ -85,9 +85,9 @@ AC_REQUIRE_AUX_FILE([tap-driver.sh])
 AC_PATH_PROG([NATURALDOCS], [naturaldocs], [notfound])
 AC_ARG_VAR([NATURALDOCS], [Path to naturaldocs])
 AC_SUBST([GLIB_COMPILE_RESOURCES], [`$PKG_CONFIG --variable glib_compile_resources gio-2.0`])
-AC_PATH_PROG([SCSS], [scss], [notfound])
-AS_IF([test x$SCSS = xnotfound], [AC_MSG_ERROR([SCSS compiler not found.
-Try installing the "sass" Ruby gem or its equivalent package.])])
+AC_PATH_PROG([SCSS], [sassc], [notfound])
```

Comment:
The problem is that the Ruby-Sass command line tool does not have the same arguments as `sassc`.

We could have two different variables and then a conditional to decide which rule to use.